build/import/Packages.targets

```diff
@@ -87,7 +87,6 @@
     <PackageReference Update="EnvDTE"                                                                 Version="16.8.30523.219" />
     <PackageReference Update="EnvDTE80"                                                               Version="16.8.30523.219" />
     <PackageReference Update="EnvDTE90"                                                               Version="16.8.30523.219" />
-    <PackageReference Update="StreamJsonRpc"                                                          Version="2.6.41-alpha" />
```

Comment:
We don't expliictly reference the `StreamJsonRpc` package anywhere in this solution, so this line had no effect.